### PR TITLE
LPS-184810 Add feature flag for LPS-87806

### PR DESCRIPTION
**Pull request by ces-quanhuynh**
Hi @tototrinh @locpham97 @ces-luannguyen 
Please help me review this PR.

### Motivation
LPS-184810 Add/Remove Feature or Release flag of LPS-154515
### Solution
Modify portal.properties in portal-impl module to show toggle box for enabling/disabling Epic feature in Instance Settings > Feature Flags. This configuration is read to enable/disable code used in epic.

Thanks,
Quan

### CODE REVIEW

File: portal-web/docroot/html/taglib/ui/input_permissions/page.jsp [hunk 1]
```diff
@@ -111,7 +111,7 @@ String permissionPropagationCheckboxLabel = (String)request.getAttribute("lifera
 					<option <%= inputPermissionsViewRole.equals(RoleConstants.OWNER) ? "selected=\"selected\"" : "" %> value="<%= RoleConstants.OWNER %>"><liferay-ui:message key="owner" /></option>
 				</select>
 
-				<c:if test="<%= Validator.isNotNull(permissionPropagationCheckboxLabel) %>">
+				<c:if test='<%= FeatureFlagManagerUtil.isEnabled("LPS-87806") && Validator.isNotNull(permissionPropagationCheckboxLabel) %>'>
```
Comment (locpham97) [inline portal-web/docroot/html/taglib/ui/input_permissions/page.jsp:114]: why you change from " to ' ?

**Comment by ces-locpham** [comment]
Change the commit to Add feature flag for the story LPS-154515 or maybe better LPS-87806

**Comment by ces-quanhuynh** [comment]
@locpham97 
Thanks, I've updated it!

**Review by ces-totrinh — APPROVED** [review]
LGTM. Thanks.

**Review by ces-locpham — APPROVED** [review]

**Comment by ces-totrinh** [comment]
Hi @hhtank, this PR passed our internal review. Please help take a look and give your feedback. Thanks.

**Comment by hhtank** [comment]
@tototrinh LGTM, thanks.

**Comment by ces-totrinh** [comment]
@ces-quanhuynh , Please help fix the conflict and update the PR. Thanks.

**Comment by ces-quanhuynh** [comment]
Thanks @tototrinh 
I've updated the PR
Quan

**Comment by ces-totrinh** [comment]
Merged. Thanks all.

**Pull request merged**